src/main/java/io/jenkins/plugins/orka/helpers/OrkaClientProxy.java

```diff
@@ -67,21 +67,31 @@ public OrkaClientProxy(String endpoint, String credentialsId, int httpClientTime
 
     public ConfigurationResponse createConfiguration(String vmName, String image, String baseImage,
             String configTemplate, int cpuCount) throws IOException {
+        return this.createConfiguration(vmName, image, baseImage, configTemplate, cpuCount, null);
+    }
+
+    public ConfigurationResponse createConfiguration(String vmName, String image, String baseImage,
+            String configTemplate, int cpuCount, String scheduler) throws IOException {
 
         try (OrkaClient client = getOrkaClient()) {
-            return client.createConfiguration(vmName, image, baseImage, configTemplate, cpuCount);
+            return client.createConfiguration(vmName, image, baseImage, configTemplate, cpuCount, scheduler);
         }
     }
 
     public DeploymentResponse deployVM(String vmName) throws IOException {
-        try (OrkaClient client = getOrkaClient()) {
-            return client.deployVM(vmName);
-        }
+        return this.deployVM(vmName, null, null);
     }
 
     public DeploymentResponse deployVM(String vmName, String node) throws IOException {
+        return this.deployVM(vmName, node, null);
+    }
+
+    public DeploymentResponse deployVM(String vmName, String node, String scheduler) throws IOException {
         try (OrkaClient client = getOrkaClient()) {
-            return client.deployVM(vmName, node);
+            if (scheduler.isEmpty()) {
```

Comment:
Do we need this here?
Ideally we would always call `client.deployVM(vmName, node, scheduler);` and it will do whatever it needs to do (check the scheduler or directly pass it)

Reply:
No. We don't need it. that is handled in the Deployment Request.